frontend/src/app/landing/LandingPage.jsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import './style/landingPage.scss';
+import TeamProfile from './TeamProfile';
+
+const LandingPage = (): React.Node => {
+    return (
+        <>
+            <div className="jumbotron text-center">
+                <h1 className="display-4">Welcome to UBB Courses!</h1>
+                <p className="lead">
+                    The aim of this project is to create a friendlier environment for the students
+                    to learn during the university.
+                </p>
+                <hr className="my-4" />
+                <br />
+                <p className="lead">
+                    <a className="btn btn-primary btn-lg" role="button" href="/courses">
+                        Explore courses
+                    </a>
+                </p>
+            </div>
+
+            <div className="text-center d-flex justify-content-center">
+                <div className="w-50">
+                    <p className="lead">About</p>
+                    <p>
+                        UBB Courses e-learning courses aim to enhance the knowledge and skills of
```

Comment:
Plese use [react bootstrap](https://react-bootstrap.github.io/components/alerts/) and try to split the code into smaller components afterwards

Reply:
Lower camel case file name